### ISSUE TITLE
1.0.6 Austin72 Aim, RCS and Trigger failing to work

### ISSUE THREAD

**Issue by AutisticFurry** [Jire/Charlatano#396]
I'm putting this here because I put it up on Austins but has had no input from others so I'll raise the question here so others using his program can get help. 

Having an issue with Flat Aim, RCS and trigger not working, even toggle set to X1 mouse not even doing anything tried middle mouse and z on my keyboard and nothing, if anyone know what's going on it would be grand if you could help

**Comment by MoofMonkey** [comment]
Does this works with defaults?
Send us your configs.

**Comment by AutisticFurry** [comment]
@MoofMonkey Everything is default, I always check the default settings before I change anything in case that's the issue but here it isn't. I'll upload my build to you and send hopefully in a private chat.

**Comment by MoofMonkey** [comment]
No, thanks. Build are everytime fine. Did you checked that using latest version of Austin72/Charlatano?

**Comment by AutisticFurry** [comment]
@MoofMonkey Yes, I made sure it was the latest and having the latest JDK, the basic Charlatano works from Jire but I am unhappy with how the aim is working so I tried Austins and it's just not working. Bhop, ESP is working so I'm kinda confused what is going on.
